pkg/controller/ovn_ic.go

```diff
@@ -553,3 +552,31 @@ func (c *Controller) listRemoteLogicalSwitchPortAddress() (*strset.Set, error) {
 
 	return existAddress, nil
 }
+
+func (c *Controller) getTSName(index int) string {
+	var tsName string
+	if index == 0 {
+		tsName = util.InterconnectionSwitch
+	} else {
+		tsName = fmt.Sprintf("%s%d", util.InterconnectionSwitch, index)
+	}
+	return tsName
+}
+
+func (c *Controller) getTSCidr(index int) (string, error) {
+	defaultSubnet, err := c.subnetsLister.Get(c.config.DefaultLogicalSwitch)
+	if err != nil {
+		return "", err
+	}
+
+	var cidr string
+	switch defaultSubnet.Spec.Protocol {
+	case kubeovnv1.ProtocolIPv4:
+		cidr = fmt.Sprintf("169.254.%d.0/24", index)
```

Comment:
Should we set `cidr = 169.254.100.0/24` when `index` is 0?

Reply:
1